pkgs/tools/audio/spotdl/default.nix

```diff
@@ -59,7 +59,9 @@ in python.pkgs.buildPythonApplication rec {
     soundcloud-v2
     bandcamp-api
     setuptools # for pkg_resources
-  ] ++ python-slugify.optional-dependencies.unidecode;
+  ] ++ python-slugify.optional-dependencies.unidecode
+    ++ rapidfuzz.optional-dependencies.full;
+    # https://github.com/NixOS/nixpkgs/pull/137296#pullrequestreview-751646500
```

Comment:
```suggestion
```
the linked comment contains no reason or explanation, just an observation. We should find something that is more useful and probably just write it verbatim in the code.

spotdl does not require the full feature of rapidfuzz and also does not mention numpy in any way. I am confused why it is required at all.

Reply:
> spotdl does not require the full feature of rapidfuzz and also does not mention numpy in any way. I am confused why it is required at all.

I am also confused! I am not a spotdl user myself, so I am only making this change to not break spotdl.

Reply:
Afaict spotdl also works without numpy. Please remove the second commit.

Reply:
Done! But I am not sure whether spotdl works without numpy; previous experience (https://github.com/NixOS/nixpkgs/pull/137296#pullrequestreview-751646500) seems to suggest it doesn't. I will try to build locally and see how it goes.

Reply:
I can confirm that spotdl builds fine on https://github.com/NixOS/nixpkgs/commit/39c91094a4dc394eac5628fc42e6d545f86b96e7. Maybe the issue is now resolved.